slave-services/src/main/java/orq/iqu/slaveservices/others/ImageServiceSlaveImpl.java

```diff
@@ -1,5 +1,19 @@
 package orq.iqu.slaveservices.others;
 
+import org.iqu.coreservices.config.ServiceInfo;
+import org.iqu.slaveservices.rest.consumer.ImageConsumer;
+import org.iqu.slaveservices.rest.consumer.factory.ConsumerFactory;
+
+import orq.iqu.slaveservices.dto.ImageDTO;
+
 public class ImageServiceSlaveImpl implements ImageServiceSlave {
 
+  private ImageConsumer imageConsumerInstance = new ConsumerFactory().getImageConsumerInstance();
```

Comment:
There's no need to create a consumerFactory object just call it directly since the methods are static: ConsumerFactory.getImageConsumerInstance();

Please replace this anywhere in the application...